packages/zoe/src/zoeService/zoe.js

```diff
@@ -209,6 +209,34 @@ function makeZoe(vatAdminSvc) {
                 brandToPurse.init(brand, E(issuer).makeEmptyPurse());
               }
             }),
+        makeEmptySeat: (keyword, initialAllocation) => {
+          let currentAllocation = initialAllocation;
+
+          const { notifier, updater } = makeNotifierKit();
+
+          const seatAdmin = {
+            replaceAllocation: replacementAllocation => {
+              harden(replacementAllocation);
+              updater.updateState(replacementAllocation);
+              currentAllocation = replacementAllocation;
+            },
+            exit: () => {
+              updater.finish(undefined);
+              const instanceAdmin = instanceToInstanceAdmin.get(instance);
+              instanceAdmin.removeZoeSeatAdmin(seatAdmin);
+
+              // burn the holdings to keep Zoe's book straight
```

Comment:
I don't think we ever want to burn the holdings - the contract or someone else probably wants them

Reply:
okay. Made available to contract.  This needs a test.